docs/flow/tutorials/chaining.md

```diff
@@ -0,0 +1,73 @@
+---
+aliases:
+- /docs/agent/latest/flow/tutorials/chaining
+title: Chaining components
+weight: 400
+---
+
+The goal of this tutorial is to show the ability to have data go to several different locations using [multiple-inputs.flow](../assets/flow_configs/multiple-inputs.flow). This uses the same base as [filtering metrics]({{< relref "filtering-metrics.md">}}).
+
+A new concept in Flow is chaining components together. 
+
+# Prerequisites
+
+* [Docker](https://www.docker.com/products/docker-desktop)
+* Clone the [Agent Repository](https://github.com/grafana/agent) `git clone git@github.com:grafana/agent.git`
+
+## Example
+
+To run the example execute `docker-compose -f ./assets/multiple-inputs.yaml up` from the tutorials directory. 
```

Comment:
nit: Doesn't make much difference, but the three docker-compose files are pretty similar, right? How about adding just one and passing the appropriate configuration file as an env var? The instruction would look like

```
CONFIG_FILE=multiple-inputs.flow docker-compose -f ./assets/agent-flow.yaml up
```

Reply:
Sounds solid, let me give it a try.

Reply:
Worked great!

Reply:
Nice! I'll keep this in mind, we might be able to reuse this elsewhere too!